governance-proposal/TSC-Project-Lifecycle.md

```diff
@@ -0,0 +1,78 @@
+## ***DRAFT for Public Comment***
+
+# Node.js Foundation Project Lifecycle
+
+Node.js Foundation’s technical Projects shall follow a lifecycle described in this Project Lifecycle document.  
+
+The TSC shall create, align and coordinate Projects that are ideally developed together, including possibly, the creation of sub-Projects. The TSC shall have the power to reorganize Projects and sub-Projects after sufficient review and discussion with the Projects and community involved.
+
+The TSC shall encourage new Projects and innovation in the technical community. New Projects enter the Node.js technical community through a Proposal to the TSC and if approved, are granted Incubation-state status.
+
+Projects shall change state following TSC reviews. Projects typically change states independently from each other, but can cooperate closely and leverage each other’s results. Projects graduate from Proposal-state, through Incubation-state and Mature-state to Core-state. Archived–state is a Project state reserved for those Projects no longer being actively developed or used by the community.
+
+The TSC shall have the authority to grandfather proposed Projects with a significant history and record to meet the project state descriptions below into a higher order state during the first 12 months after the first TSC meeting.
```

Comment:
That part hasn't quite been figured out yet. The idea that has been kicked around is that "Incubation" may occur either inside or outside of the foundation. If it's clear that the project is mature, the process could be abbreviated.